client/jira.go

```diff
@@ -120,6 +128,38 @@ func CommentIssue(issueId, comment string) (domain.Issue, error) {
 	return issue, nil
 }
 
+func Transition(issueId, transitionId string) (error, string) {
```

Comment:
return order should be reversed. Very unusual. Main thing we are trying to return is this string not error. Do we actually need this string?

Reply:
order reversed. moved to the separate file

Reply:
I've added this string in order to return the request URL (so I would be able to test the actual URL - explained in another comment dialog).

Reply:
For the order, I didn't know that the order matters. Thanks for fixing that!!